benchmarks/benchmarks/generate_data/__init__.py

```diff
@@ -92,9 +98,19 @@ def run_function_elsewhere(func_to_run, *args, **kwargs):
         f"{func_to_run.__name__}(" + ",".join(func_call_term_strings) + ")"
     )
     python_string = "\n".join([func_string, func_call_string])
-    result = run(
-        [DATA_GEN_PYTHON, "-c", python_string], capture_output=True, check=True
-    )
+
+    try:
+        result = run(
+            [DATA_GEN_PYTHON, "-c", python_string],
+            capture_output=True,
+            check=True,
+            text=True,
```

Comment:
Note the addition of the `text` argument, which removes the need for downstream `stdout.decode()` or `stderr.decode()`.